### ISSUE TITLE
add dayofweek

### ISSUE THREAD

**Issue by TheCandle** [bear-zd/ChaoXingReserveSeat#9]
Question:
I want make an reservation at a different time each day

Solution:
1. Add a field "dayofweek" in file "config.json"
2. Then in main.py,  add a funciton get_current_dayofweek to judge whether a reservation is equal to it, if not, continue

I can do it if you think it is ok.

**Comment by bear-zd** [comment]
It sounds like this task could be accomplished with crontab settings, but if you're willing to implement it, I'd welcome that as well (the current reservation logic is indeed a bit odd).
However, please make your changes on the **rebuild** branch. This branch has undergone a code refactor and new features have been added, but it hasn't been thoroughly tested yet, so it hasn't been merged into the main branch. If there are no issues later on, I'll merge the rebuild branch into the main branch.

**Comment by bear-zd** [comment]
After carefully reviewing the code, it seems that the implementation is not quite as I had imagined. It might be worth considering using a list to represent the 'dayofweek' parameter, as its functionality appears to be achievable with crontab. Additionally, it looks like for multiple day reservations within a week, the configuration needs to be copied multiple times. So I modified the code to make it more convenient.   7be0f389aa628283afe96c33a044ef88c05e9f47
